.github/workflows/verify-and-create-draft-release.yml

```diff
@@ -14,7 +14,7 @@ on:
 
 jobs:
   generate_tarballs:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
We should also update the ubuntu version in [install-and-verify.yml](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/blob/main/.github/workflows/install-and-verify.yml)